dask_expr/io/parquet.py

```diff
@@ -45,6 +47,21 @@
 _cached_plan = {}
 
 
+@normalize_token.register(pa_ds.Dataset)
+def normalize_pa_ds(ds):
+    return (ds.files, ds.schema)
+
+
+@normalize_token.register(pa_ds.FileFormat)
+def normalize_pa_file_format(file_format):
+    return str(file_format)
+
+
+@normalize_token.register(pa.Schema)
+def normalize_pa_schema(schema):
+    return schema.to_string()
```

Comment:
Cool.  This seems nicer to me.  

If it's trivial to add this to dask/dask too that maybe makes sense to do.  No pressure if you're feeling rushed though.